tmt/cli.py

```diff
@@ -388,7 +388,8 @@ def create(context, name, template, force, **kwargs):
     current working directory.
     """
     tmt.Test._save_context(context)
-    tmt.Test.create(name, template, context.obj.tree, force)
+    tmt.Test.create(name, template, context.obj.tree, force,
+                    tmt.Test._opt('dry'))
```

Comment:
This will not be necessary as the default is now handled in the method (as proposed above).